apps/browser-extension-wallet/src/hooks/useWalletManager.ts

```diff
@@ -111,13 +116,14 @@ const getHwExtendedAccountPublicKey = async (
 
 const getExtendedAccountPublicKey = async (
   wallet: AnyBip32Wallet<Wallet.WalletMetadata, Wallet.AccountMetadata>,
-  accountIndex: number
+  accountIndex: number,
+  password: string
```

Comment:
Probably doesn't matter right now due to non-strict null checks, but it makes it easier to understand the code
```suggestion
  password?: string
```

Reply:
done in 82a42907c63b1e8db4e81f2610257bd52a467436